docker-build-package

```diff
@@ -18,9 +18,10 @@ set -e
 
 IMAGE_NAME_PREFIX=registry.gitlab.com/northern.tech/mender/mender-test-containers:mender-dist-packages-builder
 
+# TODO (MEN-6395) - hardcoded to oleorhagen whilst this is a feature branch
```

Comment:
This need to refer to a future task. I created https://northerntech.atlassian.net/browse/MEN-6569.